### DISCUSSION TITLE
5c3701ff/

### DISCUSSION THREAD

**Discussion by giscus** [triccsr/triccsr.github.io#3] [Announcements]
# 5c3701ff/

摘要 3人场，7题。\n虽然我寄了，但别人也都寄了，这何尝不是一种不寄\n补题链接私信sua的人。\n题解 咕咕咕\n

https://triccsr.github.io/5c3701ff/

**Comment by wjy-yy** [comment]
Dew是个好队长（点头）

**Reply by triccsr** [reply]
活捉wjy大佬orz
